doc/man3/TS_VERIFY_CTX.pod

```diff
@@ -0,0 +1,156 @@
+=pod
+
+=head1 NAME
+
+TS_VERIFY_CTX, TS_VERIFY_CTX_new, TS_VERIFY_CTX_init, TS_VERIFY_CTX_free,
+TS_VERIFY_CTX_cleanup, TS_VERIFY_CTX_set_flags, TS_VERIFY_CTX_add_flags,
+TS_VERIFY_CTX_set0_data, TS_VERIFY_CTX_set0_imprint, TS_VERIFY_CTX_set0_store,
+TS_VERIFY_CTX_set0_certs, TS_VERIFY_CTX_set_certs, TS_VERIFY_CTS_set_certs,
+TS_VERIFY_CTX_set_data, TS_VERIFY_CTX_set_imprint, TS_VERIFY_CTX_set_store
+- manage the TS response verification context
+
+=head1 SYNOPSIS
+
+ #include <openssl/ts.h>
+
+ typedef struct TS_verify_ctx TS_VERIFY_CTX;
+
+ TS_VERIFY_CTX *TS_VERIFY_CTX_new(void);
+ void TS_VERIFY_CTX_init(TS_VERIFY_CTX *ctx);
+ void TS_VERIFY_CTX_free(TS_VERIFY_CTX *ctx);
+ void TS_VERIFY_CTX_cleanup(TS_VERIFY_CTX *ctx);
+ int TS_VERIFY_CTX_set_flags(TS_VERIFY_CTX *ctx, int f);
+ int TS_VERIFY_CTX_add_flags(TS_VERIFY_CTX *ctx, int f);
+ int TS_VERIFY_CTX_set0_data(TS_VERIFY_CTX *ctx, BIO *b);
+ int TS_VERIFY_CTX_set0_imprint(TS_VERIFY_CTX *ctx,
+                                unsigned char *hexstr, long len);
+ int TS_VERIFY_CTX_set0_store(TS_VERIFY_CTX *ctx, X509_STORE *s);
+ int TS_VERIFY_CTX_set0_certs(TS_VERIFY_CTX *ctx, STACK_OF(X509) *certs);
+
+The following functions have been deprecated since OpenSSL 3.4:
+
+ BIO *TS_VERIFY_CTX_set_data(TS_VERIFY_CTX *ctx, BIO *b);
+ unsigned char *TS_VERIFY_CTX_set_imprint(TS_VERIFY_CTX *ctx,
+                                          unsigned char *hexstr, long len);
+ X509_STORE *TS_VERIFY_CTX_set_store(TS_VERIFY_CTX *ctx, X509_STORE *s);
+ STACK_OF(X509) *TS_VERIFY_CTX_set_certs(TS_VERIFY_CTX *ctx,
+                                         STACK_OF(X509) *certs);
+
+The following function has been deprecated since OpenSSL 3.0:
+
+ STACK_OF(X509) *TS_VERIFY_CTS_set_certs(TS_VERIFY_CTX *ctx,
+                                         STACK_OF(X509) *certs);
+
+
+=head1 DESCRIPTION
+
+The Time-Stamp Protocol (TSP) is defined by RFC 3161. TSP is a protocol used to
+provide long-term proof of the existence of certain data before a particular
+time. TSP defines a Time Stamping Authority (TSA) and an entity that makes
+requests to the TSA. Usually, the TSA is referred to as the server side, and the
+requesting entity is referred to as the client.
+
+In TSP, when a server sends a response to a client, the server normally
+needs to sign the response data - the TimeStampToken (TST) - with its private
+key. Then the client verifies the received TST using the server's certificate
+chain.
+
+For all the following methods, unless noted otherwise, B<ctx> is the
```

Comment:
I know you're just copying the page however as you're already touching this, could you please use `I<ctx>` and similarly elsewhere throughout the manpage where names of arguments are referenced?